src/js/visual/config.part.js

```diff
@@ -79,7 +97,7 @@ ripe.Config.prototype.update = function(state, options) {
     // if the parts and the position haven't changed
     // since the last frame load then ignores the
     // load request and returns immediately
-    var size = this.element.getAttribute("data-current-size");
+    var size = this.element.dataset.current_size;

```

Comment:
the convention is to still use camel case when accessing `dataset` attributes, or am I missing some gotcha for a particular browser?

Reply:
The thing is internally we often use the nomenclature 'data-something_else'.
Previously, on the code, we had all data-attribute names separated by hyphen ('data-current-size') so we updated them to the hyphen/underscore version ('data-current_size'). 

Should we change it?

Reply:
By the way, when we perform "element.dataset.currenSize" the result will be "data-current-size".

Reply:
Just as a side note I was wrong and the `-` character is allowed in the data attributes, but let's just keep using the `_` character instead https://stackoverflow.com/questions/33068777/what-characters-are-allowed-not-allowed-in-the-names-of-custom-data-attribut